spring-cloud-contract-verifier/src/test/groovy/org/springframework/cloud/contract/verifier/builder/MockMvcMethodBodyBuilderSpec.groovy

```diff
@@ -559,7 +559,7 @@ class MockMvcMethodBodyBuilderSpec extends Specification implements WireMockStub
 		builder.appendTo(blockBuilder)
 		def test = blockBuilder.toString()
 		then:
-		test.contains('get("/users?limit=10&offset=20&filter=email&sort=name&search=55&age=99&name=Denis.Stepanov&email=bob@email.com")')
+		test.contains('get("/users?limit={limit}&offset={offset}&filter={filter}&sort={sort}&search={search}&age={age}&name={name}&email={email}","10","20","email","name","55","99","Denis.Stepanov","bob@email.com")')
```

Comment:
you made me think, maybe it would be even better to use this method instead?
.queryParam(parameterName, parameterValues)

and generate something like this:

```

ResponseOptions response = given().spec(request).queryParam("filter", "{\"___any___\":\"9999\"}")
					.get("/incomingFiles");

```